src/components/30-organisms/footer/story.js

```diff
@@ -0,0 +1,48 @@
+/* global document */
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from '@storybook/html';
+import './index';
+import { withMarkdown } from '../../../../.storybook/addons/markdown';
+// import withBodyReset from '../../../../.storybook/addons/reset-body';
```

Comment:
don't leave commented-out code in, version control handles that

Reply:
Why no nobodyReset?? Footer would be broken without